app/models/movie.rb

```diff
@@ -1,3 +1,9 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def average_age 
+    actors.average(:age).round(0)
+  end
```

Comment:
Depending on who you talk to you are going to get differing opinions on the `.round` that you are doing here and whether this is presentation logic or not and therefore belongs in the view. I believe that this is presentation logic because you are changing the way the data is displayed and comes back. So instead of a decimal with many places you are truncating it to an integer. This is all for display.

If in fact, you wanted to display the original number from the database, you would then need to create another method to do that instead of being able to just use this one.

I think it's much easier to do this `.round` in the view. and in your test, you can do the same thing: `expect(@movie.average_age.round(0)).to eq(...)`